docs/_Aug_milestone_.md

```diff
@@ -0,0 +1,46 @@
+## August Milestone
+
+### Overview
+
+- Finish hasManyThrough relation MVP
+- Continue with docs improvements
+
+### Goals
+
+- [ ] hasManyThrough relation MVP
+
+  - [ ] docs: add HasManyThrough relation #5835
+  - [ ] feat: set up HasManyThrough relation through the CLI #5836
+
+- [ ] Documentation improvement
+
+  - [ ] docs: improve the structure of "Behind the scenes" sidebar + find a
+        better name #5776
+  - [ ] docs: improve the structure of "Usage scenarios" (how-to guides) sidebar
+        #5777
+  - [ ] [Spike] docs: untangle connector reference into reference, how-to guides
+        and tutorials #5961
+  - [ ] Rename "legacy juggler bridge" mentioned in docs #5558
+  - [ ] Add Couchdb connector docs #5267
+  - [ ] [APIDocs] Table not showing properly when it includes code snippet #5756
+
+- [ ] PRs to review:
+  - [ ] feat(authentication-jwt) Add signup / create user #5763
+  - [ ] feat(authentication): add support for multiple strategies on same method
+        #5735
+  - [ ] 5187 jwt refresh token Feature #5589
+  - [ ] [RFC WIP] feat(typegoose): add a component for typegoose/mongoose
+        integration #5152
+
+### Stretch Goals (Community contribution is welcome!)
+
```

Comment:
@dhmlau The authorization doc improvement is ready to consider https://github.com/strongloop/loopback-next/issues/5441#issuecomment-663108274.

Reply:
we might. @agnes512 has 2 August tasks that have PRs under review.

Reply:
@jannyHou, I added #5441 to stretch goal so that we can get the bug in. Thanks.